tensorflow/python/ops/nn_fused_batchnorm_test.py

```diff
@@ -377,6 +379,8 @@ def testInferenceShape4(self):
             x_shape, dtype, [6], np.float32, use_gpu=True, data_format='NHWC')
       self._test_inference(
           x_shape, dtype, [6], np.float32, use_gpu=False, data_format='NHWC')
+      self._test_inference(
```

Comment:
For consistency with the GPU case, you should put the NCHW case first. And same for some of the test cases below